Entity added, properties and first methods added

azzilian opened the pull request:
Pom - added dependencies for springboot,mysql,lombok
application.properties - added properties to connect to DB OnlineBookStoreApplication - added simple test
Book - entity added
BookService + BookRepository - methods save + findAll ad

azzilian committed:
Pom - added dependencies for springboot,mysql,lombok
application.properties - added properties to connect to DB
OnlineBookStoreApplication - added simple test
Book - entity added
BookService + BookRepository - methods save + finAll ad

andrii-hoienko reviewed (changes requested):
Good job, but some improvements required

azzilian committed:
BookRepository - empty lines removed
BookRepositoryImpl - proper exception used now
Book - removed annotations for size of columns
application.properties - l & p removed

azzilian committed:
test application.properties - l & p removed

andrii-hoienko reviewed (approved):
Good job!
